### DISCUSSION TITLE
DSC 10 and DSC 40A at UC San Diego

### DISCUSSION THREAD

**Discussion by surajrampure** [kevinlin1/just-the-class#14] [Show and tell]
https://dsc10.com
https://dsc40a.com

**Comment by dhilanr** [comment]


**Reply by kevinlin1** [reply]
You can find them in the `gh-pages` branch of the repositories managed by [dsc-courses](https://github.com/dsc-courses).
- [dsc10-2022-sp](https://github.com/dsc-courses/dsc10-2022-sp/tree/gh-pages)
- [dsc40a-2022-wi](https://github.com/dsc-courses/dsc40a-2022-wi/tree/gh-pages)

**Comment by surajrampure** [comment]
Yep - thanks, Kevin!
